scalatest/src/main/scala/org/scalatest/concurrent/PimpedThreadGroup.scala

```diff
@@ -74,7 +74,7 @@ private[concurrent] class PimpedThreadGroup(threadGroup: ThreadGroup) {
   def getThreads(recursive: Boolean): List[Thread] = {
     def getThreads(sizeEstimate: Int): Seq[Thread] = {
       val ths = new Array[Thread](sizeEstimate)
-      if (threadGroup.enumerate(ths, recursive) == sizeEstimate) getThreads(sizeEstimate + 10)
+      if (threadGroup.enumerate(ths.asInstanceOf[Array[Thread | Null]], recursive) == sizeEstimate) getThreads(sizeEstimate + 10)
```

Comment:
**Pass array into java method**
Pass `Array[T]` as argument to java method that requires an array. Java arrays have type `Array[T | Null]`.